nodejs/express-redis/app/package.json

```
@@ -0,0 +1,19 @@
+{
+  "name": "express-redis",
```

```suggestion
  "name": "express-redis",
  "private": true,
```

No accidental pushes to npm